src/site/includes/header.html

```diff
@@ -23,6 +23,20 @@
 <meta name="insight-app-sec-validation" content="ec59f99e-d869-4e3f-82d2-a61d32c65776">
 {% endif %}
 
+{% assign analytics = buildtype | default: 'vagovprod' %}
+{% if analytics == 'localhost' %}{% assign analytics = 'vagovdev' %}{% endif %}
+{% assign analytics = analytics | prepend: "src/site/assets/google-analytics/" %}
+{% assign analyticsScript = analytics | append: ".js" %}
+{% assign analyticsIframe = analytics | append: ".html" %}
```

Comment:
We create paths to the GA files in `src/site/assets/google-analytics`, then just include them in the page.